server/core/src/main/java/dev/slimevr/protocol/datafeed/DataFeedHandler.kt

```diff
@@ -0,0 +1,196 @@
+package dev.slimevr.protocol.datafeed
+
+import com.google.flatbuffers.FlatBufferBuilder
+import dev.slimevr.protocol.DataFeed
+import dev.slimevr.protocol.GenericConnection
+import dev.slimevr.protocol.ProtocolAPI
+import dev.slimevr.protocol.ProtocolAPIServer
+import dev.slimevr.protocol.ProtocolHandler
+import dev.slimevr.protocol.datafeed.DataFeedBuilder.createDevicesData
+import dev.slimevr.protocol.datafeed.DataFeedBuilder.createServerGuard
+import dev.slimevr.protocol.datafeed.DataFeedBuilder.createStayAlignedPose
+import dev.slimevr.protocol.datafeed.DataFeedBuilder.createSyntheticTrackersData
+import dev.slimevr.tracking.trackers.Tracker
+import io.eiren.util.logging.LogManager
+import solarxr_protocol.MessageBundle
+import solarxr_protocol.data_feed.DataFeedConfigT
+import solarxr_protocol.data_feed.DataFeedMessage
+import solarxr_protocol.data_feed.DataFeedMessageHeader
+import solarxr_protocol.data_feed.DataFeedUpdate
+import solarxr_protocol.data_feed.PollDataFeed
+import solarxr_protocol.data_feed.StartDataFeed
+import java.util.function.Consumer
+import java.util.stream.Collectors
+
+class DataFeedHandler(private val api: ProtocolAPI) : ProtocolHandler<DataFeedMessageHeader>() {
+	init {
+		registerPacketListener(
+			DataFeedMessage.StartDataFeed,
+		) { conn: GenericConnection?, header: DataFeedMessageHeader? ->
```

Comment:
```registerPacketListener(DataFeedMessage.StartDataFeed, ::onStartDataFeed)```

You should be able to replace all these with this syntax instead. I know we didnt do it on some file either. But this really is the better syntax. Lets switch to that 🙏